packages/components/src/Notification/ToastNotification/ToastNotification.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useId } from "react"
+import { addToastNotification } from "./subcomponents/ToastNotificationManager"
+import { ToastNotificationWithOptionals } from "./types"
+
+export type ToastNotificationProps = Omit<
+  ToastNotificationWithOptionals,
+  "message" | "persistent"
+> & {
+  /**
+   * Removes the dismiss trigger. functions the same as `persistent` in `addToastNotification`. If this is true you will need to manage the removal of notifications manually.
+   * @default false
+   */
+  hideCloseIcon?: boolean
```

Comment:
Functionally this does the same thing as persistent, so I've updated the type for the omit that from the ToastNotificationProps so there's less likelihood someone will pass in both into the component